doc/labels.xml

```diff
@@ -73,6 +73,10 @@ gap> DigraphVertexLabel(D, 1);
     in <A>digraph</A> to the list of
     arbitrary &GAP; objects <A>list</A>. <P/>
 
+    If the list <A>list</A> is immutable, then the vertex labels are set to a
+    mutable copy of <A>list</A>. Otherwise, the labels are set to exactly
+    <A>list</A>. <P/>
+
```

Comment:
While you're at it here, could you specific that `<A>list</A>` (when it is introduced) must be the same length of the number of vertices of `<A>digraph</A>`, and also below, when it talks about `<C>vertices</C>`, you could note the analogous thing about mutable copies, there (if indeed a mutable copy is being made in that case; if it's not then it should be!)